doc/cprover-manual/contracts-invariant.md

```diff
@@ -0,0 +1,123 @@
+# Invariant Clause
+
+An _invariant_ clause specifies a property that must be preserved
+by every iteration of a loop.
+
+### Syntax
+
+An _invariant_ clause accepts a valid Boolean expression
+over the variables visible at the same scope as the loop.
+Additionally, [history variables] are also allowed within invariant clauses.
```

Comment:
That bit about "history variables" looks like it should be a link to a history variables page, but it is being rendered just as "history variables" in square brackets, which reads oddly.

Reply:
Ah yes! Good catch; thanks. Fixed it -- added a link to the history variables Markdown file.